openai/openai_test.go

```diff
@@ -404,7 +386,6 @@ func TestMiddlewareResponses(t *testing.T) {
 		TestPath string
 		Handler  func() gin.HandlerFunc
 		Endpoint func(c *gin.Context)
-		Setup    func(t *testing.T, req *http.Request)
 		Expected func(t *testing.T, resp *httptest.ResponseRecorder)
```

Comment:
This PR is great as-is, however we should also do the same for the model list / retrieve handlers.